pandas/core/groupby/generic.py

```diff
@@ -377,8 +383,8 @@ def _get_index() -> Index:
             result = Series(data=values, index=_get_index(), name=self._selection_name)
             return self._reindex_output(result)
 
-    def _aggregate_named(self, func, *args, **kwargs):
-        result = OrderedDict()
+    def _aggregate_named(self, func, *args, **kwargs) -> dict:
```

Comment:
Can you use `typing.Dict` instead of the built-in? Would also be great if you can identify key / value types if not too much incremental effort

Reply:
Will do.  In cases where we can do `Dict[foo, bar]` i get why that's better.  In cases where we can't say anything meaningful, is it just for consistency or is there some other upside?